pilot/cmd/pilot-discovery/main.go

```diff
@@ -127,11 +127,11 @@ func init() {
 		"comma separated list of networking plugins to enable")
 
 	// MCP client flags
-	discoveryCmd.PersistentFlags().IntVar(&serverArgs.MCPMaxMessageSize, "mcpMaxMsgSize", defaultMCPMaxMessageSize,
```

Comment:
I was wondering when I cleaned this stuff last week, do we need these `MCPOptions` to be configurable or defaults are sufficient?

Reply:
max message size is definitely often used since if you hit the limit your entire cluster blows up and no config is sent

Reply:
Ok. Is there any downside of setting it to a very high default or no limit? Does server also have similar/same value configured?

Reply:
Good point, I am not sure if its impacting buffer sizes or anything... we are no longer using galley mcp anymore though so its a bit less important